languages/clojure/config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "language": "Clojure",
+  "active": true,
+  "blurb": "a JVM hosted functional programming language.",
+  "version": 1,
```

Comment:
```suggestion
  "version": 3,
```

Reply:
@ErikSchierboom thank you for pointing this out, actually i'm not sure what this version is for, is it for clojure or clojure in exercism? If it is the former, I see clojure version is now at 1.10.*, please correct me if wrong:-)

Reply:
It is the version of the configuration format, which is co-incidentally equal to the version of exercism its targetting.

Reply:
@NobbZ got it, corrected, thank u~